sktime/forecasting/statsforecast.py

```diff
@@ -628,14 +644,165 @@ def get_test_params(cls, parameter_set="default"):
             `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
             `create_test_instance` uses the first (or only) dictionary in `params`
         """
-        from statsforecast.models import AutoETS
+        from sktime.forecasting.theta import ThetaForecaster
 
         del parameter_set  # to avoid being detected as unused by ``vulture`` etc.
 
-        params = [
-            {}, 
-            {"season_length": 4, "trend_forecaster": AutoETS(model="ZZN")},
-            {"season_length": 4, "trend_forecaster": AutoETS(model="ZZN"), "stl_kwargs": {"robust": True}}
-        ]
+        params = {
+            "season_length": [3, 12],
+            "trend_forecaster": ThetaForecaster(),
+        }
+
+        return params
+
+
+class StatsForecastBackAdapterMSTL(BaseForecaster):
+    """StatsForecast Back Adapter for StatsForecast MSTL.
+
+    StatsForecastBackAdapter is a wrapper for
+
+    Parameters
+    ----------
+    estimator :
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.statsforecast import StatsForecastMSTL
+    >>> from sktime.forecasting.statsforecast import StatsForecastBackAdapterMSTL
+    >>> from sktime.forecasting.theta import ThetaForecaster
+
+    >>> y = load_airline()
+    >>> trend_forecaster = StatsForecastBackAdapterMSTL(ThetaForecaster())
+    >>> model = StatsForecastMSTL(
+            season_length=[3,12],
+            trend_forecaster=trend_forecaster
+        )
+    >>> fitted_model = model.fit(y=y)
+    >>> y_pred = fitted_model.predict(fh=[1,2,3])
+    >>> y_pred
+    1961-01    454.507079
+    1961-02    419.341442
+    1961-03    477.373918
+    Freq: M, Name: Number of airline passengers, dtype: float64
+    """
+    def __init__(self, estimator):
+        self.estimator = estimator
+        self._forecaster = None
+        self._model = None
+        self.in_sample_fh = None
+        self.in_sample_X = None
+
+        super().__init__()
+
+        if hasattr(self.estimator, "seasonal"):
+            if self.estimator.seasonal:
+                raise Exception(
+                    "Trend forecaster should not adjust seasonal models."
+                )
+
+        if hasattr(self.estimator, "sp"):
+            if self.estimator.sp > 1:
+                raise Exception(
+                    "Trend forecaster should not adjust seasonal models. Please pass "
+                    "`sp=1` to your trend forecaster"
+                )
+
+    def __repr__(self):
+        return "StatsForecastBackAdapterMSTL"
+
+    def fit(self, y, X=None):
+        self.estimator.fit(y=y, X=X)
+        self.in_sample_fh = -np.arange(len(y))
```

Comment:
Here I've set `in_sample_fh` to values [-len(y)...0] to represent in-sample forecast horizon